README.md

```diff
@@ -18,3 +19,63 @@ pip install -e .
 GITHUB_TOKEN=... review-gator --config branches.yaml
 ```
 
+Basic Configuration
+-------------------
+
+Configuration is provided via a YAML file. To see an example, run 
+`review-gator --config-skeleton`
+
+```
+launchpad:
+    owners:
+        # Launchpad username
+        LAUNCHPAD-USERNAME:
+            max-age: 30
+    branches:
+        # Full path to repo
+        LAUNCHPAD-BZR-REPO-PATH:
+            review-count: 2
+github:
+    repos:
+        GITHUB-USERNAME:
+            GITHUB-PROJECT-NAME:
+                review-count: 2
+lp-git:
+    owners:
+        LAUNCHPAD-USERNAME:
+            max-age: 30
+    repos:
+        LAUNCHPAD-GIT-REPO-PATH:
+            review-count: 2
+```
+NOTE: as of v. 0.2, `owners` and all options underneath are experimental 
+with no guarantee of working. If you'd like to improve this work, we'll happily
+take PRs!
```

Comment:
'owners' was added 3 years ago (prior to v0.2) in commit 6edf6350.  What is the rationale for making this experimental? Was there some discussion of this policy?  I know that we've been using this in production for some time now.  Because I know that we've been using the feature forever the 2nd sentence reads as "if it's broken why don't you fix my code" and it rubs me the wrong way.

Reply:
Had IRC conversation with @rcj4747 . Discussed the issues around the `owner` stanza and the best way of communicating the issue. At present `get_branches_for_owner` isn't working as expected. I'll be changing this to be clear that we are aware the capabilities aren't as we'd like, that it requires  more testing and development, and will add TODO items to the README